.env.example

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_FIREBASE_USE_EMULATOR=1
```

Comment:
would `NEXT_PUBLIC_FIREBASE_USE_EMULATOR=true` be more intuitive?

Reply:
Agreed — updated.